apps/meteor/app/lib/server/functions/setUserActiveStatus.ts

```diff
@@ -90,10 +90,6 @@ export async function setUserActiveStatus(userId: string, active: boolean, confi
 
 	await Users.setUserActive(userId, active);
 
-	if (active && !user.active) {
-		await callbacks.run('afterActivateUser', user);
```

Comment:
Hey @MarcosSpessatto , When I removed this part of code. I could not find any other use case of 'afterActivateUser' callback.
![afterActivateUser](https://github.com/RocketChat/Rocket.Chat/assets/15830206/3f7e4e98-393f-4135-9319-41dcad1dccb3)
Did a search on this callback, and found only one usecase (which i removed).
So I thought of completely removing this callback. Am I missing something :thinking: ?

Reply:
No, you are right, if it's not being used anymore, we can remove that. I was talking about the other behaviors, like, previously whenever the agent changed its status the callback was dispatched and the `await LivechatTyped.addAgent(user.username)` was called, inside this method we do have some logic and by removing this we also would not execute that code anymore, I'm just asking if losing those behaviors is the expected behavior since we can have some collateral effects by doing that.

Reply:
I believe this is a good idea. 👍